packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/key-message.twig

```diff
@@ -55,9 +55,14 @@
 </script>
 {% endif %}
 
-<section {% if id %}id="{{ id }}" {% endif %}class="ma__key-message{{ msgClass }}{% if bgImage %} ma__key-message--image{% endif %}{% if keyMessage.textOverlay and bgImage %} ma__key-message--image-overlay{% endif %}{% if positionTop %} ma__key-message--top{% endif %}"{% if bgImage %} style="background-image: url('{{ bgImage }}')"{% endif %}>
-  {% if bgImage and positionTop %}
-    <div id="img{{ id }}"  class="ma__key-message__inline-image" {% if keyMessage.mobileBgImage and positionTop %} style="background-image: url('{{ keyMessage.mobileBgImage }}')"{% endif %}>
+<section {% if id %}id="{{ id }}" {% endif %}class="ma__key-message{{ msgClass }}{% if bgImage %} ma__key-message--image{% endif %}{% if keyMessage.textOverlay and bgImage %} ma__key-message--image-overlay{% endif %}{% if positionTop %} ma__key-message--top{% endif %}">
+  {% if bgImage%}
+    <div id="img{{ id }}"  class="ma__key-message__inline-image">
+      {% if bgImage.path %}
+        {% include bgImage.path with bgImage.data %}
+      {% else %}
+        {{ bgImage }}
```

Comment:
is this how the responsive `picture` element is passed in from the drupal responsive image module?  If so, can you add a comment in the check to explain why this is added for future reference? Thanks!

Reply:
I added comments in the template to document the purpose of those two lines.